Cargo.toml

```diff
@@ -13,6 +13,7 @@ edition = "2018"
 [dependencies]
 parking_lot_core = { path = "core", version = "0.7.1" }
 lock_api = { path = "lock_api", version = "0.3.4" }
+cfg-if = "0.1"
```

Comment:
This is not needed. `core/Cargo.toml` already has a dependency on `cfg-if`.

Reply:
Sorry, did not see that.